<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>751</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Menú Principal</string>
     </attribute>
     <widget class="QLabel" name="label1">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>160</y>
        <width>451</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>LABORATORIO #7</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>240</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>MENÚ</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ejercicio #1</string>
     </attribute>
     <widget class="QLabel" name="label1_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>451</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>EJERCICIO #1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>671</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_Nombre">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>140</y>
        <width>431</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Apellido:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_Apellido">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>200</y>
        <width>431</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nota:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SPB_Nota">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>260</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>310</y>
        <width>671</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="botonAgregarEstudiante">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>360</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Agregar Estudiante</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>661</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>A continuación, ingrese los datos del estudiante.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>350</y>
        <width>3</width>
        <height>61</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="botonReporte_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>370</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Acciones Adicionales</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>340</y>
        <width>331</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Crear y leer archivos. Además, crear reportes.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Ejercicio #2</string>
     </attribute>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>671</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label1_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>451</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>EJERCICIO #2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>200</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Código:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="SPB_Codigo">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>200</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="botonActualizar">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>300</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Actualizar</string>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>671</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="botonCargarProductos">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>420</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Cargar Archivo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botonCargarProductos_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>420</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Archivo de Prueba</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Ejercicio #3</string>
     </attribute>
     <widget class="Line" name="line_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>671</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label1_4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>451</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>EJERCICIO #3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="botonGenerar3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>200</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Generar Archivo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botonEstadisticas">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>320</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Generar Estadísticas</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botonCargarArchivo">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>260</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Cargar Archivo</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
